### PR TITLE
Update lstm_seq2seq.py(from 22% to 87% acc)

### PR DESCRIPTION
**Pull request by tykimos**
I added the codes for applying one-hot encoding on the end of sentences about encoder_input_data, decoder_input_data, and decoder_target_data. I added an accuracy metric for model training. The original code has 22% accuracy, but the proposed code had 87% validation accuracy.

### Summary
Applying one-hot encoding on the end of sentences

### Related Issues
#13266

### PR Overview

- [n] This PR requires new unit tests [y/n] (make sure tests are included)
- [n] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [y] This PR is backwards compatible [y/n]
- [n] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by nuxlear** [comment]
@tykimos I think it should be written with PEP 8 convention. Please check Travis-CI results for which codes make the failure. 
https://travis-ci.org/keras-team/keras/jobs/578675382

**Comment by tykimos** [comment]
> @tykimos I think it should be written with PEP 8 convention. Please check Travis-CI results for which codes make the failure. You can find details on line 1308.
> 
> https://travis-ci.org/keras-team/keras/jobs/578675382

**Comment by tykimos** [comment]
> @tykimos I think it should be written with PEP 8 convention. Please check Travis-CI results for which codes make the failure. You can find details on line 1308.
> 
> https://travis-ci.org/keras-team/keras/jobs/578675382

Thanks. I committed updated code according to PEP 8.

**Comment by tykimos** [comment]
@abhinavsagar Thanks to your approval. What is the next step for this PR?

**Comment by abhinavsagar** [comment]
Cc @fchollet
